src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -393,23 +393,23 @@ msgstr "Live Z Anpassen"
 #. Ignore
 #: src/gui/dialogs/dialog_response.cpp:17
 msgid "LOAD"
-msgstr "Laden"
+msgstr "LADEN"
 
 #: src/gui/dialogs/DialogFactory.cpp:13 src/gui/screen_menu_filament.cpp:52
 msgid "LOAD FILAMENT"
-msgstr "Filament laden"
+msgstr "FILAMENT LADEN"
 
 #: src/gui/screen_menu_filament.cpp:51
 msgid "Load Filament"
 msgstr "Filament laden"
 
 #: src/gui/screen_menu_lan_settings.cpp:202
 msgid "Load Settings"
-msgstr "Ladeeinstellungen"
+msgstr "Einstellungen laden"
 
 #: src/gui/screen_splash.cpp:43
 msgid "Loading ..."
-msgstr "Lade ..."
+msgstr "Lade..."
```

Comment:
There should be a blank bevor ellipsis: It is not a sign like comma, it represents some words and then there has to be a blank. I learned this also just a year ago :)